src/types/autocomplete.d.ts

```diff
@@ -12,7 +12,10 @@ export default Autocomplete;
 export interface AutocompleteParameters {
   numResults?: number;
   filters?: Record<string, any>;
+  filtersPerSection?: Record<string, Record<string, any>>;
```

Comment:
Not the most familiar with typescript, but should we restrict this to `Record<string, Record<string, string[]>>`?